packages/base/src/toolbar/widget.tsx

```diff
@@ -38,66 +38,69 @@ export class ToolbarWidget extends ReactiveToolbar {
     this.addClass('jGIS-toolbar-widget');
 
     if (options.commands) {
-      this.addItem(
-        'undo',
-        new CommandToolbarButton({
-          id: CommandIDs.undo,
-          label: '',
-          icon: undoIcon,
-          commands: options.commands
-        })
-      );
+      const undoButton = new CommandToolbarButton({
+        id: CommandIDs.undo,
+        label: '',
+        icon: undoIcon,
+        commands: options.commands
+      });
 
-      this.addItem(
-        'redo',
-        new CommandToolbarButton({
-          id: CommandIDs.redo,
-          label: '',
-          icon: redoIcon,
-          commands: options.commands
-        })
-      );
+      this.addItem('undo', undoButton);
+      undoButton.node.dataset.testid = 'undo-button-test-id';
+
+      const redoButton = new CommandToolbarButton({
+        id: CommandIDs.redo,
+        label: '',
+        icon: redoIcon,
+        commands: options.commands
+      });
+      this.addItem('redo', redoButton);
 
       this.addItem('separator0', new Separator());
 
-      this.addItem(
-        'Toggle console',
-        new CommandToolbarButton({
-          id: CommandIDs.toggleConsole,
-          commands: options.commands,
-          label: '',
-          icon: terminalToolbarIcon
-        })
-      );
+      const toggleConsoleButton = new CommandToolbarButton({
+        id: CommandIDs.toggleConsole,
+        commands: options.commands,
+        label: '',
+        icon: terminalToolbarIcon
+      });
+      this.addItem('Toggle console', toggleConsoleButton);
+      toggleConsoleButton.node.dataset.testid = 'toggle-console-button-test-id';
```

Comment:
```suggestion
      toggleConsoleButton.node.dataset.testid = 'toggle-console-button';
```

The dataset stuff adds an attribute like `data-testid=value` to the element, so having that be like `data-testid="undo-button-test-id"` seems unnecessarily verbose

Reply:
@gjmooney Thanks for your comment. I have removed the `-test-id` suffix in all `data-testid` values.